### PR TITLE
common/LogEntry: include EntityName in log entries

### PR DESCRIPTION
**Pull request by liewegas**
See the entity logger authenticated as (mon.$hostname, client.foo, mgr.$hostname) instead
of the entity_name_t (client.4122, mon.0).  This is particularly helpful for librbd
users and for ceph-mgr, which otherwise appears as client.NNN in the logs.

### CODE REVIEW

File: src/common/LogClient.cc [hunk 1]
```diff
@@ -342,6 +343,11 @@ const entity_inst_t& LogClient::get_myinst()
   return messenger->get_myinst();
 }
 
+const EntityName& LogClient::get_myname()
```
Comment (tchaikov) [inline src/common/LogClient.cc]: could be a `const` method.

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-02_08:32:01-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**